spec/controllers/meetings_controller_spec.rb

```diff
@@ -109,6 +109,15 @@
       expect(response).to have_http_status(:ok)
     end
 
+    it 'joins as moderator if "glAnyoneJoinAsModerator" setting enabled' do
+      allow_any_instance_of(Room).to receive(:anyone_joins_as_moderator?).and_return(true)
+      room = create(:room, user:, viewer_access_code: 'AAA', moderator_access_code: 'BBB')
+
+      expect_any_instance_of(BigBlueButtonApi).to receive(:join_meeting).with(room:, name: 'Someone', role: 'Moderator')
+      get :join, params: { friendly_id: room.friendly_id, name: 'Someone', access_code: 'AAA' }
+      expect(response).to have_http_status(:ok)
+    end
```

Comment:
I would add another test case where there is no access code

Reply:
To explicitly ensure that passing `access_code` doesn't change the role to viewer?